cli/sub_command.go

```diff
@@ -73,6 +92,10 @@ func configureSubCommand(app commandHost) {
 	act.Flag("raw", "Show the raw data received").Short('r').UnNegatableBoolVar(&c.raw)
 	act.Flag("translate", "Translate the message data by running it through the given command before output").StringVar(&c.translate)
 	act.Flag("ack", "Acknowledge JetStream message that have the correct metadata").BoolVar(&c.jsAck)
+	// We do not support (explicit) ackPolicy right now. The only situation where it is useful would be WorkQueue policy right now.
+	// Deleting from a stream with WorkQueue through ack could be unexpected behaviour in the sub command.
```

Comment:
need to use US spelling for behaviour :)

Reply:
Sure